doc/nrf/known_issues.rst

```diff
@@ -2691,6 +2691,13 @@ NCSDK-17501: Partition Manager: Ignored alignment for partitions
 
   **Workaround:** Edit the affected configurations file so that ``align`` is correctly placed inside the ``placement`` section of the config file.
 
+.. rst-class:: v2-2-0
+
+NCSDK-19536: Boards without a uart1 node do not compile with TF-M logging enabled
+  TF-M will not compile when the board does not have a uart1 node defined in device tree configuration.
```

Comment:
```suggestion
  TF-M will not compile when the board does not have a uart1 node defined in the devicetree configuration.
```

Reply:
It might be just me, but I find it strange that the title says that the boards won't compile with TF-M enabled, and the text for the known issue says that TF-M won't compile (because the board is missing a uart1 node). Should we align these two? I don't know if it matters, I just thought I'd point it out :)

Reply:
> It might be just me, but I find it strange that the title says that the boards won't compile with TF-M enabled, and the text for the known issue says that TF-M won't compile (because the board is missing a uart1 node). Should we align these two? I don't know if it matters, I just thought I'd point it out :)

I have rephrased, the sentences was a bit strange I agree. Hopefully this is better.